terra-aou-rstudio/rstudio/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e # catches errors
+
+# Restrict uploads to 4096 kbit/s (0.512 MB/s) and unlimited download
+tc qdisc add dev eth0 root tbf rate 4096kbit latency 50ms burst 1540000
```

Comment:
also please double-check that end-user is unable to call `tc`

Reply:
Removed permissions to even view `tc` and others in `sbin`

```
rstudio@aou:~$ tc
bash: tc: command not found
rstudio@aou:~$ speedtest
Retrieving speedtest.net configuration...
Testing from Google Cloud (34.135.35.254)...
Retrieving speedtest.net server list...
Selecting best server based on ping...
Hosted by Binary.Net (Omaha, NE) [12.44 km]: 29.904 ms
Testing download speed................................................................................
Download: 2045.38 Mbit/s
Testing upload speed......................................................................................................
Upload: 5.71 Mbit/s
```

Reply:
cannot even do `/sbin/tc` right?